js/commonjs/rewrite_tests_for_commonjs.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @fileoverview Utility to translate test files to CommonJS imports.
```

Comment:
The existing tests use this approach, and this PR is not removing the existing tests. This PR is adding new functionality and new tests for native CommonJS functionality that doesn't depend on `goog.require()` or `deps.js`.